index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
 <title>Apache Druid (incubating)</title>
-
+<link rel="canonical" href="https://druid.apache.org/index.html" />
```

Comment:
This should be https://druid.apache.org/ (no index.html)

Reply:
I mean, either one would work, but https://druid.apache.org/ is prettier -- which is the point of canonical URLs.

Reply:
Fixed, thanks